src/app/app.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="page-wrap">
-  <h1 class="party-payments-header" [title]="'Click to copy the URL'" (click)="copyLink()">Party Payments</h1>
+  <h1 class="party-payments-header" [title]="'Party Payments'">Party Payments</h1>
```

Comment:
### 🤓 Nitpick: remove title

If we don't want to copy the link by click - we should remove the title either.